### PR TITLE
Alpha cleanupwarnings

### PR DESCRIPTION
**Pull request by Wagnerp**
* Cleanup some warnings

### CODE REVIEW

**Comment by Wagnerp** [comment]
@Smurf-IV Is this ok to be merged?

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Calendar/Classes/CalendarDay.cs [hunk 1]
```diff
@@ -295,7 +295,10 @@ internal void UpdateHighlights()
 
             for (int i = 0; i < TimeUnits.Length; i++)
             {
-                TimeUnits[i].SetHighlighted(TimeUnits[i].CheckHighlighted());
+                if (TimeUnits != null)
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Calendar/Classes/CalendarDay.cs:298]: Move if statement before for loop.

**Review by Smurf-IV — CHANGES_REQUESTED** [review]

**Comment by Smurf-IV** [comment]
> @Smurf-IV Is this ok to be merged?

Sorry, the review was not "Posted"..

**Comment by Wagnerp** [comment]
> > @Smurf-IV Is this ok to be merged?
> 
> Sorry, the review was not "Posted"..

No problem :)

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Calendar/Classes/CalendarDay.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Calendar/Classes/CalendarDay.cs]: Now moved

**Pull request merged**